src/Liquidator.sol

```diff
@@ -214,267 +224,304 @@ contract Liquidator is Owned {
 
-        emit AuctionStarted(account, trustedCreditor, baseCurrency, uint128(openDebt));
+        // Emit event
+        emit AuctionStarted(account, creditor, uint128(debt));
     }
 
     /**
-     * @notice Function returns the current auction price of a Account.
-     * @param account The contract address of the Account.
-     * @return price the total price for which the Account can be purchased.
-     * @return inAuction returns false when the Account is not being auctioned.
-     * @dev We use a dutch auction: price constantly decreases and the first bidder buys the Account
-     * and immediately ends the auction.
+     * @notice Calculate the starting price for a liquidation based on the specified debt amount.
+     * @param debt The amount of debt for which to calculate the starting price.
+     * @return startPrice The calculated starting price, expressed as a fraction of the debt.
+     * @dev This function is an internal view function, and it calculates the starting price for a liquidation
+     *      based on a given debt amount. The start price is determined by multiplying the debt by the
+     *      `startPriceMultiplier` and dividing the result by 100.
      */
-    function getPriceOfAccount(address account) public view returns (uint256 price, bool inAuction) {
-        inAuction = auctionInformation[account].inAuction;
-
-        if (!inAuction) {
-            return (0, false);
-        }
-
-        price = _calcPriceOfAccount(auctionInformation[account]);
+    function _calculateStartPrice(uint256 debt, uint256 startPriceMultiplier_)
+        internal
+        pure
+        returns (uint256 startPrice)
+    {
+        startPrice = debt * startPriceMultiplier_ / 100;
     }
 
     /**
-     * @notice Function returns the current auction price given time passed and the openDebt.
-     * @param auctionInfo The auction information.
-     * @return price The total price for which the Account can be purchased.
-     * @dev We use a dutch auction: price constantly decreases and the first bidder buys the Account and immediately ends the auction.
-     * @dev Price P(t) decreases exponentially over time: P(t) = openDebt * [(SPM - MPM) * base^t + MPM]:
-     * SPM: The startPriceMultiplier defines the initial price: P(0) = openDebt * SPM (2 decimals precision).
-     * MPM: The minPriceMultiplier defines the asymptotic end price for P(∞) = openDebt * MPM (2 decimals precision).
-     * base: defines how fast the exponential curve decreases (18 decimals precision).
-     * t: time passed since start auction (in seconds, 18 decimals precision).
-     * @dev LogExpMath was made in solidity 0.7, where operations were unchecked.
+     * @notice Calculate asset distribution percentages based on provided risk values.
+     * @param riskValues_ An array of risk values for assets.
+     * @return assetDistributions An array of asset distribution percentages (in tenths of a percent, e.g., 1_000_000 represents 100%).
```

Comment:
ten thousands of a percentage?

Reply:
If max value is 1m, could also use uint24. Not sure of gas savings in memory though